docker-for-windows/faqs.md

```diff
@@ -239,13 +146,11 @@ Getting Started topic. For more about Docker Machine itself, see
 [What is Docker Machine?](/machine/overview#what-is-docker-machine), and the
 [Hyper-V driver](/machine/drivers/hyper-v) for Docker Machine.
 
-## Windows Requirements
+## Windows requirements
 
 ### How do I run Windows containers on Docker Desktop on Windows Server?
 
-See [About Windows containers and Windows Server](/install/windows/docker-ee/#about-docker-ee-containers-and-windows-server).
-
-A full tutorial is available in [docker/labs](https://github.com/docker/labs){: target="_blank" class="_"} at
+A tutorial about running Windows containers is available in [docker/labs](https://github.com/docker/labs){: target="_blank" class="_"} at
```

Comment:
You can find a tutorial about running Windows containers on Windows Server in [Getting Started with Windows Containers](https://github.com/docker/labs/blob/master/windows/windows-containers/README.md){: target="_blank" class="_"}.

Reply:
Fixed.